docs/examples/frameworks/jax/flax-basic_example.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Suggestions:</p><p>"For DALI pipeline to work with JAX it needs to be wrapped with appropriate DALI iterator." -&gt; "For DALI pipeline to work with JAX, the former needs to be wrapped with an appropriate DALI iterator."</p><p> </p><p>"In addition to the pipeline we can pass the" -&gt; "In addition to the DALI pipeline object we can pass the"</p><p> </p><p>"<strong>Here is a quick explnation of how these parameters work:" </strong>- maybe it would be good to put a link to the full documentation somewhere here?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done